jammsen/docker-palworld-dedicated-server
# Add Player logging

Resolves #216 

Adding in logging as a background task instead of using supercronic so we don't need to create temporary files

## Test instructions

1. Start a server
2. Verify it logs it you leaving and joining in docker
3. With players in game test with others joining
4. Test multiple players joining within the same polling period
5. Delete player data, create a new players with any name that may break it (Spaces, commas, numbers, etc)
